src/main/java/io/pakland/mdas/githubstats/application/SaveCommits.java

```diff
@@ -0,0 +1,21 @@
+package io.pakland.mdas.githubstats.application;
+
+import io.pakland.mdas.githubstats.domain.Commit;
+import io.pakland.mdas.githubstats.domain.repository.CommitRepository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+public class SaveCommits {
+
```

Comment:
I would also change the name to `SaveAllCommits` which is the naming we are usign, as in #83

Reply:
Didn't notice that, but it makes sense to add the keyword All when there is no partial list save? To my taste isn't needed because SaveCommits is enough descriptive. The All part seems coupled to the name of JPA method saveAll, but in the first place, this method could be named simply "save" and be overloaded with different params, a list in this case.

Reply:
Hmm, okey, convincing enough.